base/custom_metrics.go

```diff
@@ -71,42 +72,45 @@ type Params struct {
 	Value string `json:"value" yaml:"value"`
 }
 
-// collectDatabaseInputs is the input to the collect-metrics-database task
-type collectDatabaseInputs struct {
-	// Providers is the set of labels/names of the data sources
-	Providers []string `json:"providers" yaml:"providers"`
+// customMetricsInputs is the input to the custommetrics task
+type customMetricsInputs struct {
+	// ProviderURLs a slice of URLs for metric templates
+	ProviderURLs []string `json:"providerURLs" yaml:"providerURLs"`
```

Comment:
Provider templates are now hosted in URLs instead of supplied as files (within charts). This means any user can simply put their template in any URL (e.g. their GH repo) and start using it in Iter8 experiments.